src/store/flow/block-types/MobilePrimitives_OpenResponseBlockStore.ts

```diff
@@ -24,22 +25,23 @@ const actions: ActionTree<IEmptyState, IRootState> = {
 
```

Comment:
We can alternatively:
- use `block_updateConfigByPath` instead of `block_updateConfigByKey` to change individual config values directly
- remove return

Reply:
Good catch